cypress/e2e/pages/homePage/urls.js

```diff
@@ -1,4 +1,11 @@
 const urls = [
+  {
+    // put in alphabetical order when not testing
+    service: 'portuguese',
+    local: '/portuguese',
+    test: '/portuguese?renderer_env=test',
```

Comment:
We can remove the `renderer_env` bit as that is only used for getting the media player to playback correctly on Test.

Reply:
Done, and put it into alphabetical order